fuzzbench/run_experiment.py

```diff
@@ -46,8 +48,15 @@ def run_experiment():
 
     while True:
         print('Current status of jobs:')
+        print('\tqueued:\t%d' % len(queue))
```

Comment:
consider using either `.count` or `len()` everywhere for consistency

Reply:
Good suggestions! I use this form from the documentation. I'm not sure they share a same way for requesting length/amount. Will check it later.

Reply:
Done. We use `.count` for consistency.